docs/index.md

```
@@ -195,6 +195,34 @@ After installation the binaries and Python modules live inside the environment,
 conda activate combine
 ```
 
+##### Standalone compilation with `pixi` (CMake-based)
+If you have access to CVMFS (for example on `lxplus`), a ready-to-use installation of `pixi` is available and can be set up by sourcing:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)